vizkit3d_world.orogen

```diff
@@ -23,6 +23,16 @@ task_context "Task" do
     property("world_file_path", "/std/string")
 
     #
+    # set the viewport width
+    #
+    property("viewport_width", "int", 800)
```

Comment:
Just call it "width" and "height" since it is the size of the generated image. The fact that there is a viewport is an implementation detail.